Makefile

```diff
@@ -88,20 +93,23 @@ diagram:
 #######################
 
 test:
-	go test ./...
+	# Angus: The Go race detector is an incredible tool. It will crash the program and warn you if it detects a data
```

Comment:
Oh nice, I will always use it now for tests and local development. Thank you for the tips! 😀